### PR TITLE
Melittapirjolea branch

### PR DESCRIPTION
**Pull request by MelittaPirjolea**
added Melitta Pirjolea tasks

### CODE REVIEW

File: melitta-pirjolea-dir/melittapirjolea.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Melitta Pirjolea created this file during lab5
+second commit
```
Comment (BadeaCristina) [inline melitta-pirjolea-dir/melittapirjolea.txt:2]: reviewed by CristinaBadea

**Review by BadeaCristina — COMMENTED** [review]
Finish review

File: melitta-pirjolea-dir/melittapirjolea.txt (hunk 1 above)
Comment (MelittaPirjolea) [reply melitta-pirjolea-dir/melittapirjolea.txt:2]: Done

**Review by BadeaCristina — APPROVED** [review]
done

File: melitta-pirjolea-dir/melittapirjolea.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+reviewed by CristinaBadea
```
Comment (ionut0) [inline melitta-pirjolea-dir/melittapirjolea.txt:3]: Please add "reviewed by Ionut Poplacean"

**Review by ionut0 — CHANGES_REQUESTED** [review]

File: melitta-pirjolea-dir/melittapirjolea.txt (hunk 2 above)
Comment (MelittaPirjolea) [reply melitta-pirjolea-dir/melittapirjolea.txt:3]: Done

**Review by ionut0 — APPROVED** [review]

**Pull request merged**